packages/react-shopify-app-route-propagator/README.md

```diff
@@ -65,74 +30,135 @@ export interface Props {
 }
 ```
 
-Allowing you to use it with a string
+### `useRoutePropagator`
 
-```javascript
-<ShopifyRoutePropagator location="/foo/bar?thing=true" app={app} />
-```
-
-Or a location object
+This example uses [app bridge](https://help.shopify.com/en/api/embedded-apps/app-bridge#set-up-your-app) to create an app instance and `withRouter` HoC form `react-router-router`.
```

Comment:
> instance and `withRouter` HoC form `react-router-router`

should be

> instance and the `withRouter` enhancer from `react-router`

(enhancer or higher-order-component are preffered over acronyms)